backend/app.js

```diff
@@ -4,6 +4,8 @@ var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 var cors = require("cors");
+var mongoose = require('mongoose');
```

Comment:
Can these all be changed to cost instead of var, since they won't be changed? Can you do it for all the variables at the top of this file (I know you didnt write a lot of these variables, but I'm just noticing this now).